app/bank/application/reset_funds_amount/reset_funds_amount_command_handler.py

```diff
@@ -0,0 +1,31 @@
+from bank.application.reset_funds_amount.reset_funds_amount_command import ResetFundsAmountCommand
+from bank.domain.account_repository import AccountRepository
+from bank.domain.account_updater import AccountUpdater
+from bank.domain.exceptions.account_movements.account_has_no_movements_exception import AccountHasNoMovementsException
+from bank.domain.historic_movement_repository import HistoricMovementRepository
+from cqrs.commands.command_handler import CommandHandler
+
+
+class ResetFundsAmountCommandHandler(CommandHandler):
+    def __init__(self, historic_movement_repository: HistoricMovementRepository, account_repository: AccountRepository, account_updater: AccountUpdater):
+        self.__historic_movement_repository = historic_movement_repository
+        self.__account_repository = account_repository
+        self.__account_updater = account_updater
+
+    def handle(self, command: ResetFundsAmountCommand):
+        all_accounts = self.__account_repository.filter_accounts()
+        for account in all_accounts:
+           if all_accounts == 0:
+               raise AccountHasNoMovementsException()
```

Comment:
this exception will be raised when there are no accounts in the Repository

I am 100% sure that you do not want to implement this